petclinic-frontend/src/features/customers/api/updateOwner.ts

```diff
@@ -13,7 +13,5 @@ export const updateOwner = async (
 export const getOwner = async (
   userId: string
 ): Promise<AxiosResponse<OwnerResponseModel>> => {
-  return await axios.get<OwnerResponseModel>(
-    `http://localhost:8080/api/gateway/owners/${userId}`
-  );
+  return await axiosInstance.get<OwnerResponseModel>(`owners/${userId}`);
```

Comment:
Could you provide an explanation on why you changed to axiosInstance?

Reply:
> Could you provide an explanation on why you changed to axiosInstance?

Using the shared axiosInstance ensures consistency since everyone will be using it. Also, the request will trigger the axiosErrorResponseHandler that will redirect users to the apprioriate error pages.